sci-libs/umfpack/umfpack-5.6.2.ebuild

```diff
@@ -14,11 +14,13 @@ SLOT="0"
 KEYWORDS="~alpha amd64 ~arm arm64 hppa ~ia64 ~mips ppc ppc64 sparc x86 ~amd64-linux ~x86-linux ~x86-macos"
 IUSE="doc +cholmod static-libs"
 
+# doesn't build against cholmod-3.0.13 and amd-2.4.6
 RDEPEND="
 	>=sci-libs/amd-1.3
+	<sci-libs/amd-2.4.6
 	sci-libs/suitesparseconfig
 	virtual/blas
-	cholmod? ( >=sci-libs/cholmod-2[-minimal] )"
+	cholmod? ( >=sci-libs/cholmod-2[-minimal(-)] <sci-libs/cholmod-3 )"
```

Comment:
this line makes no sense.

Reply:
why? Should i just use the `<sci-libs/cholmod-3` term?

Reply:
yes of course